src/reducers/gui.js

```diff
@@ -102,13 +99,24 @@ const initTutorialLibrary = function (currentState) {
         currentState,
         {
             modals: {
-                previewInfo: false,
                 tipsLibrary: true
             }
         }
     );
 };
 
+const initPreviewInfo = function (currentState) {
```

Comment:
I'm not crazy about loading up gui.js with these mini-reducers that mutate state in these arbitrary ways... could it be possible to put this in a reducer? Maybe the modals reducer?

Is this necessary basically because we want this to be the starting state on construct, rather than waiting for a dispatch?